app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -612,6 +612,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
+    var shouldShowOpenInAppBanner by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_user_knows_about_pwa),
```

Comment:
Based on the specs I don't think this new banner should be tied to PWAs.
Maybe we could use a new counterPreference (with a new preferenceKey) like just below of this with a similar `maxCount = 1`

// I see there is already a `pref_key_should_show_open_in_app_banner`

Reply:
It was forgot there from previous tests. 👍

Reply:
I think a simple Boolean preference will suffice, because banners are mostly shown for a first time occurrence.